### PR TITLE
[#85996930] Remove unused buttons from option panel

### PR DESCRIPTION
**Pull request by ArthurZaharov**
https://www.pivotaltracker.com/story/show/85996930

### CODE REVIEW

File: assets/javascripts/discourse/templates/post-part.hbs [hunk 1]
```diff
@@ -18,7 +18,7 @@
       </div>
     </div>
 
-    <div class='topic-body'>
+    <div class='topic-body part-view'>
```
Comment (VladimirMikhailov) [inline assets/javascripts/discourse/templates/post-part.hbs]: why won't remove them at all?

Comment (ArthurZaharov) [inline assets/javascripts/discourse/templates/post-part.hbs]: To remove button at all I must redefine post-menu view. It is more complicated. I have to do that?

Comment (VladimirMikhailov) [inline assets/javascripts/discourse/templates/post-part.hbs]: sounds strange, but yes, it's more appreciated way, please try to remove it. And if it take more then one hour, please move forward with this purpose. Thanks

**Comment by VladimirMikhailov** [comment]
:+1:

**Pull request merged**